src/zoom.js

```diff
@@ -341,14 +340,24 @@ export default function() {
     }
     else if (g.touch0) p = g.touch0[0], l = g.touch0[1];
     else return;
-    g.zoom("touch", constrain(translate(t, p, l), g.extent, translateExtent));
+
+    var move = constrain(translate(t, p, l), g.extent, translateExtent);
+
+    if ((move.x != 0 || move.y != 0) && !touchScroll) {
```

Comment:
`touchScroll` we need to prevent the noevent() when you are already scrolling the page, the browser ingores it anyways